cip-0000/cip-0000.md

```diff
@@ -0,0 +1,392 @@
+
+The CIP editor will:
+
+* Assign a CIP number in the pull request.
+
+* Merge the pull request when it is ready.
+
+
+The CIP editors are intended to fulfill administrative and editorial responsibilities. The CIP editors monitor CIP changes, and update CIP headers as appropriate.
+
+CIP editors may also, at their option, unilaterally make and merge strictly-editorial changes to CIPs, such as correcting misspellings, fixing broken links, etc.
+
+## CIP format
+
+CIPs should be written in markdown format.
+
+Each CIP should have the following parts:
+
+* Preamble -- Headers containing metadata about the CIP ([[#CIP header preamble|see below]]).
+
+* Abstract -- A short (~200 word) description of the issue being addressed.
+
+* Specification -- The specification should describe the governance change, tokenomics change and impact, and/or the syntax and semantics of any new feature. The specification should be detailed enough to allow Super Validator node operators to determine whether a given implemnetation or on-chain vote proposal matches the specification. 
+
+* Motivation -- The motivation is critical for CIPs that want to change the Global Synchronizer protocol. Changes to goverrnance or tokenonimcs should also provide a motivation. Motivations should clearly explain why the existing protocol is inadequate to address the problem that the CIP solves, or why existing governance or tokenomics are inadequate.
+
+* Rationale -- The rationale fleshes out the specification by describing what motivated the proposal and why particular decisions were made. In the case of software designs, the rationale should describe alternate designs that were considered, and related work. The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.
+
+* Backwards compatibility -- All CIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The CIP must explain how the author proposes to deal with these incompatibilities.
+
+* Reference implementation -- Required for changes to Daml models and the Canton protocol. This reference implementation must be completed before any Standards-track CIP is given status "Final", but it need not be completed before the CIP is accepted. It is better to finish the specification and rationale first and reach consensus on it before writing code. The final implementation must include test code and documentation appropriate for the Global Synchronizer.
+
+* Copyright -- The CIP must be explicitly licensed under acceptable copyright terms ([see below](#cip-licensing)). 
+
+The template for these cips is located at [CIP-XXXX](/cip-XXXX).
+
+### CIP header preamble
+
+Each CIP must begin with an RFC 822 style header preamble. The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
+
+<pre>
```

Comment:
based on this the following fields seem to be mandatory:
comments-uri, license.

However, the preamble of this CIP does not include that.

It seems to make like comments-uri maybe should not be required, at least it's not clear to me what I should set that to for a new CIP. license makes sense to require but then we should probably also add it to this CIP here.

Reply:
Thanks! I changed the [template](https://github.com/global-synchronizer-foundation/cips/pull/4/files) to have comments-uri not required and added the license to this CIP.